src/VisualStudio/CSharp/Impl/Options/Formatting/NewLinesViewModel.cs

```diff
@@ -21,6 +21,13 @@ class C {
         private static string s_methodPreview = @"class c {
 //[
     void Foo(){
+        int bar = 42;
+
+        int LocalFunction(int x) {
+            return 2 * x;
+        }
+
+        var baz = 9.75;
     }
```

Comment:
This preview text should be localized (see the other viewmodel classes).  Also, we're supporsed to avoid identifiers like `bar` and `baz`.